InventoryInStorePickup/Model/SearchRequest/Area/GetDistanceToSources.php

```diff
@@ -47,22 +48,30 @@ class GetDistanceToSources
      */
     private $searchTermPipeline;
 
+    /**
+     * @var LoggerInterface
+     */
+    private $logger;
+
     /**
      * @param GetLatsLngsFromAddressInterface $getLatsLngsFromAddress
      * @param GetOrderedDistanceToSources $getOrderedDistanceToSources
      * @param AddressInterfaceFactory $addressInterfaceFactory
      * @param Pipeline $searchTermPipeline
+     * @param Logger $logger
      */
     public function __construct(
         GetLatsLngsFromAddressInterface $getLatsLngsFromAddress,
         GetOrderedDistanceToSources $getOrderedDistanceToSources,
         AddressInterfaceFactory $addressInterfaceFactory,
-        Pipeline $searchTermPipeline
+        Pipeline $searchTermPipeline,
+        LoggerInterface $logger
```

Comment:
$logger should be null by default
```suggestion
        LoggerInterface $logger=null
```